src/TextInput/TextInput.tsx

```diff
@@ -60,7 +61,7 @@ export const TextInput = forwardRef(function TextInput(
       outlined={outlined}
       value={value}
     >
-      <>
+      <Box flex>
```

Comment:
this is needed to fix the trailing icon that's now broken because we remove `flex` from the field content